packages/material-ui/src/Grid/Grid.js

```diff
@@ -171,24 +172,24 @@ export const styles = (theme) => ({
   'align-content-xs-space-around': {
     alignContent: 'space-around',
   },
-  /* Styles applied to the root element if `justify="center"`. */
-  'justify-xs-center': {
+  /* Styles applied to the root element if `justifyContent="center"`. */
+  'justify-content-xs-center': {
```

Comment:
Isn't this a breaking change?

Reply:
In theory yes, in practice, is there a use case for using these class names? If you feel that it's too risky, happy to take a step back. I went with the fast approach so far.

Reply:
No, I think it's pretty safe. I ignored a similar change in another deprecation PR review.